include/mruby.h

```diff
@@ -1305,32 +1305,29 @@ MRB_API mrb_value mrb_vformat(mrb_state *mrb, const char *format, va_list ap);
 */
 #ifdef MRB_SYM
```

Comment:
If `mruby/presym.h` is not included **before** `mruby.h`, `MRB_SYM` is not defined here, so `mrb_intern_lit()` version seems to be used everytime. Am I missing something?